### PR TITLE
CAMS-582 extension form field width

### PR DESCRIPTION
**Pull request by fmaddenflx**
# Purpose

Fixed Trustee Form related interface issues.

# Major Changes

Completely redesigned form screen and related scss.

# Testing/Validation

Form coverage should be close to 100% and also improved a lot of coverage on form fields, including date fields.

# Definition of Done:

- [ ] Code refactored for clarity: Developers can understand the work simply by reviewing the code
- [ ] Dependency rule followed: More important code doesn’t directly depend on less important code
- [ ] Development debt eliminated: UX and code aligns to the team’s latest understanding of the domain
- [ ] No regressions: Changes do not cause regression in related or unrelated areas of the application

### CODE REVIEW

File: user-interface/src/App.scss [hunk 1]
```diff
@@ -59,8 +61,9 @@ body {
     flex: 1;
     padding: 0 0 50px 0;
     margin: 0 auto;
-    width: 83.3333333333333%; // 10/12 * 100 to stay consistent with previous 1 + 10 + 1 layout
-    max-width: 82rem;
+    width: min(83.3333333333333%, 82rem); /* Use min() for better responsive behavior */
+    max-width: calc(100vw - 2rem); /* Ensure we never exceed viewport minus padding */
+    box-sizing: border-box;
```
Comment (fmaddenflx) [inline user-interface/src/App.scss:66]: remove comments

File: user-interface/src/lib/components/uswds/DateRangePicker.test.tsx [hunk 2]
```diff
@@ -154,4 +154,458 @@ describe('Test DateRangePicker component', async () => {
...
+    const { formatDateForVoiceOver } = await import('./DateRangePicker');
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const result = formatDateForVoiceOver(malformedDate as any);
+
+    // Should return undefined due to catch block
+    expect(result).toBeUndefined();
+  });
+
+  it('should execute debounced clearValue calls in clearValue method', () => {
```
Comment (fmaddenflx) [inline user-interface/src/lib/components/uswds/DateRangePicker.test.tsx:568]: Don't use it().  use test()

File: user-interface/src/lib/components/uswds/DateRangePicker.test.tsx [hunk 3]
```diff
@@ -154,4 +154,458 @@ describe('Test DateRangePicker component', async () => {
...
+
+    // Manually trigger the callback to execute the code inside debounce
+    const callback = setTimeoutSpy.mock.calls[0][0] as () => void;
+    callback();
+
+    // Clean up
+    setTimeoutSpy.mockRestore();
+  });
+
+  it('should handle edge case in formatDateForVoiceOver with invalid date components', async () => {
```
Comment (fmaddenflx) [inline user-interface/src/lib/components/uswds/DateRangePicker.test.tsx:600]: Don't use it().  use test()
